src/shared/consts.tsx

```diff
@@ -51,3 +54,25 @@ export const enum Roles {
 }
 
 export const SEARCH_DEBOUNCE_VALUE = 700;
+
+export const itemsTypeIcons = {
```

Comment:
can you move  `itemsTypeIcons` to shared const, becasuse we have the same const in the file  src/modules/Builder/features/ActivityItems/ActivityItems.const.tsx